package.json

```diff
@@ -5,7 +5,7 @@
   "homepage": "https://aframe.io/",
   "main": "dist/aframe-master.js",
   "scripts": {
-    "dev": "cross-env INSPECTOR_VERSION=dev webpack serve --port 8080",
+    "dev": "cross-env INSPECTOR_VERSION=dev webpack serve --port 4000",
```

Comment:
I guess you committed that change by mistake.

Reply:
Oops.. :sweat_smile: